### ISSUE TITLE
Document hstore handling

### ISSUE THREAD

**Issue by pnorman** [terranodo/tegola#340]
https://github.com/terranodo/tegola/blob/22685cae2b913c002a003671fdfaabdff16eeb60/provider/postgis/util.go#L170-L185

This section of code has special handling for hstore, but it's not documented anywhere.

**Comment by Hoovs** [comment]
Looks like https://github.com/go-spatial/tegola/blob/master/provider/postgis/util.go#L170-L185 is now no longer there, or anywhere in that file.

**Comment by ARolek** [comment]
@Hoovs good catch. The hstore datatype is now handled by `case map[string]pgtype.Text:`
